Paintroid/src/main/java/org/catrobat/paintroid/listener/DrawingSurfaceListener.kt

```diff
@@ -154,7 +165,7 @@ open class DrawingSurfaceListener(
                 if (eventTouchPoint.x < drawerEdgeSize || view.getWidth() - eventTouchPoint.x < drawerEdgeSize) {
                     return false
                 }
-                timerStartDraw = System.currentTimeMillis()
```

Comment:
this got removed so currently draw tools do not smooth anymore.

Reply:
This was not intended, thank you for pointing it out.